tests/drivers/spi/spi_loopback/src/spi.c

```diff
@@ -449,9 +450,9 @@ static int spi_rx_every_4(struct spi_dt_spec *spec)
 
 static int spi_rx_bigger_than_tx(struct spi_dt_spec *spec)
 {
-	const uint32_t tx_buf_size = 8;
+	const uint32_t tx_buf_size = TX_BUF_SIZE;
 
-	BUILD_ASSERT(tx_buf_size < BUF_SIZE,
+	BUILD_ASSERT(TX_BUF_SIZE < BUF_SIZE,
```

Comment:
maybe move this build assert to be right after the macro definitions instead of in this function

Reply:
good idea, done